### ISSUE TITLE
[FEATURE REQUEST] Connect to multiple OBS Websockets at once

### ISSUE THREAD

**Issue by TheSwerik** [rivafarabi/deckboard#553]
**Is your feature request related to a problem? Please describe.**
I use 2 instances of OBS Studio (one for recording and one for streaming).
So I can only use the websocket buttons for one of them and regular hotkeys (F13 - F24) for the other.
When more buttons get a toggle state (like recording, streaming, etc.) I can only have the toggles for one of my instances.

**Describe the solution you'd like**
Let me connect to multiple OBS websockets at the same time (different ports)

**Describe alternatives you've considered**
Let me choose Toggle for regular hotkeys aswell
It's not the best option but if connecting to multiple OBS websockets is too much work or impossible then this would be a great substitute.

**Comment by grevius5** [comment]
I think you can use only one instance of obs to stream and record at the same time

**Comment by TheSwerik** [comment]
No im currently using multiple instances, one for streaming and one for recording at the same time

I'm using it so it is possible :D

**Comment by XsFrozdY** [comment]
You could just use one instance of OBS, there's a handy "Profile" and "Scene Collection" selection at the top of OBS, you can use either one.

Just make a streaming "Profile" or "Scene Collection" and a recording "Profile" or "Scene Collection".

Solved... Deckboard will automatically update for it as well.

**Comment by TheSwerik** [comment]
The thing is that I am Streamin and recording at the same time with different resolutions and FPS-settings and Scenes

**Comment by TheSwerik** [comment]
So my feature request is that you can connect to multiple OBS Websockets and maybe select for each Button which Socket should be targeted
Maybe even an option to target all websockets simultaneously

Yes you can run multiple instances of OBS at the same time
Yes you can have multiple websockets open if the ports are different
No I don't see any way to not use 2 Instances of OBS for my specific usecase

**Comment by XsFrozdY** [comment]
> The thing is that I am Streaming and recording at the same time with different resolutions and FPS-settings and Scenes
> 
> That's why I need 2 Instances at the same time

Yes, that's what you have "profiles" for... it changes EVERYTHING, settings, scenes, EVERYTHING...

**Comment by TheSwerik** [comment]
> Yes, that's what you have "profiles" for... it changes EVERYTHING, settings, scenes, EVERYTHING...

That makes no sense. I said I use both at the same time. I want 2 different resolutions and FPS-settings and Scenes at the same time/simultaneously.
One setting is for streaming and one for recording. But I do both at the same time.

**Comment by TheSwerik** [comment]
Sorry that I haven't answered for so long. I forgot about this issue when I had exams and Github didn't notify me.

I still want this feature.
Yes, this is possible. I can prove it if you want.
Yes, this is an actual use case and no I cannot do my recording and streaming differently.

The only working workaround, for now, is to use streamlabs OBS and OBS Studio together. But I don't like Streamlabs OBS.
That's why I'm requesting to add the option for multiple OBS Websockets to connect to and then select the OBS Websocket you want in the Macro (for example in "Toggle Recording").
